### PR TITLE
Iss39 create selector for institutions

### PR DESCRIPTION
**Pull request by MiguelOcegueraM**
# Description
We implemented a method for each users with the aim taht we hide functions when they're not necessary (some buttons) also we did a filter in the places by each institute
 
 # Testing
- Clone the repo in your computer
- Run rails s
- enjoy!
 
  ## Screenshots 

image

### CODE REVIEW

**Review by fego-dev — APPROVED** [review]
Good job.

File: config/database.yml [hunk 1]
```diff
@@ -16,7 +16,10 @@ default: &default
   
 development:
   <<: *default
-
+  host: localhost
+  username: root
+  password: toor
```
Comment (OchoaHub) [inline config/database.yml:21]: Nice password

**Review by jnegrete15 — APPROVED** [review]

File: test/.http [hunk 2]
```diff
@@ -67,3 +67,6 @@ Authorization: Bearer eyJhbGciOiJIUzI1NiJ9.eyJqdGkiOiIxZjIyN2U4MS1jMDFkLTQ2NzAtO
 
 ### login jazbeen
```
Comment (OchoaHub) [inline test/.http:68]: LA AMOOO

**Review by OchoaHub — APPROVED** [review]

File: test/.http (hunk 2 above)
Comment (MiguelOcegueraM) [reply test/.http:68]: 5mentarios

File: .ruby-version [hunk 3]
```diff
@@ -1 +1 @@
-ruby-2.7.0
+ruby-2.7.2
```
Comment (EduardoGHdez) [inline .ruby-version]: Was it really needed? 🤔

File: test/.http (hunk 2 above)
Comment (EduardoGHdez) [reply test/.http:68]: ❓

Comment (EduardoGHdez) [reply test/.http:68]: :baia_baia:

**Pull request merged**